src/node_data.js

```diff
@@ -127,10 +127,15 @@ const getData = function(node) {
  * @return {!NodeData} The NodeData for this Node.
  */
 const importNode = function(node) {
+  let data = node['__incrementalDOMData'];
```

Comment:
Yep. Alternatively just get rid of getData since it does the same thing as import node (or just alias it).